sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Utils.java

```diff
@@ -144,4 +144,8 @@ static String getTableName(@NonNull TokenResult tokenResult) {
     static String getUserTableName(String accountId) {
         return String.format(USER_TABLE_FORMAT, accountId).replace("-", "");
     }
+
+    static String getOutgoingId(String partition, String documentId, String pendingOperation) {
```

Comment:
there is only 1 pending operation per document, you need only partition and documentId